backend/routes/indexer.go

```diff
@@ -165,6 +168,51 @@ func processPixelPlacedEvent(event IndexerEvent, w http.ResponseWriter) {
 	WriteResultJson(w, "Pixel placement indexed successfully")
 }
 
+func processExtraPixelsPlacedEvent(event IndexerEvent, w http.ResponseWriter) {
+	placedByAddress := event.Event.Keys[1][2:]
+	positionsLenHex := event.Event.Data[0]
+
+	positionsLen, err := strconv.ParseInt(positionsLenHex, 0, 64)
+	if err != nil {
+		WriteErrorJson(w, http.StatusInternalServerError, "Error converting positionsLen hex to int")
+		return
+	}
+
+	colorsLenHex := event.Event.Data[1+positionsLen]
+	colorsLen, err := strconv.ParseInt(colorsLenHex, 0, 64)
+	if err != nil {
+		WriteErrorJson(w, http.StatusInternalServerError, "Error converting colorsLen hex to int")
+		return
+	}
+	if colorsLen != positionsLen {
+		WriteErrorJson(w, http.StatusBadRequest, "Colors length does not match positions length")
+		return
+	}
+
+	// Get Address extra pixels in postgres
+	var available, used int64
+	err = core.ArtPeaceBackend.Databases.Postgres.QueryRow(context.Background(),
+		"SELECT available, used FROM ExtraPixels WHERE address = $1", placedByAddress).Scan(&available, &used)
+	if err != nil {
+		WriteErrorJson(w, http.StatusInternalServerError, "Error getting ExtraPixels into postgres")
+		return
+	}
+
+	// Set new extra pixels in postgres
+	new_available := available - positionsLen
+	new_used := used + positionsLen
+	if new_available < 0 {
+		WriteErrorJson(w, http.StatusBadRequest, "Not enough available extra pixels")
+		return
+	}
+
+	_, err = core.ArtPeaceBackend.Databases.Postgres.Exec(context.Background(), "UPDATE ExtraPixels SET available = $1, used = $2 WHERE address = $3", new_available, new_used, placedByAddress)
```

Comment:
Its better to use a single query, instead of getting and setting from 2 seperate calls. You can do something like this here instead : https://stackoverflow.com/questions/10233298/how-to-increment-value-in-postgresql